news/2019-03-10-project-loved-week-of-march-10th.md

```diff
@@ -0,0 +1,207 @@
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+osu! Loved candidates were chosen by [-Roxas](https://osu.ppy.sh/users/1986262), [BeasttrollMC](https://osu.ppy.sh/users/3171691), [Bubbleman](https://osu.ppy.sh/users/5182050), [DigitalHypno](https://osu.ppy.sh/users/4384207) and [waywern2012](https://osu.ppy.sh/users/5870453)!
+
+---
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/ao-infinity.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Seiryu - AO-Infinity](https://osu.ppy.sh/beatmapsets/595127#osu) by [Mordred](https://osu.ppy.sh/users/7265097) and [Telomere](https://osu.ppy.sh/users/6295380)  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+A map almost lost to time resurfaces to participate in this week's Project Loved voting! Some may recognize this song from a previous Loved map by [RyuK](https://osu.ppy.sh/users/6304246), however this mapset by Mordred is considered by many to be the first successful and widely accepted rendition of Seiryu's AO-Infinity. This set comes with two difficulties: Vill's EXHAUST at 6.52 stars, and Mordred's INFINITY at 7.09 stars. Each difficulty does a great job at emphasizing the streamy and jumpy nature of the song and creates a perfect environment for players looking for a challenge on some good tunes.
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/error.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Aimiya Zero - -ERROR](https://osu.ppy.sh/beatmapsets/426602#osu) by [Yusomi](https://osu.ppy.sh/users/4174940)  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+Known for her high difficulty and jump-based maps, Yusomi returns again with her take on Aimiya Zero's "-ERROR". The song tells a story of a character drowning from their emotions after a heartbreaking separation. Although trivial, the mapset does a clean job of simulating a similar experience. Instead of emotions, any player who attempts this map will find themselves drowning in a barrage of increasingly difficult 200 BPM jumps and bursts. I think it's a fitting theme for the song. The map itself has amassed over 110 favorites since its submission in March of 2016, and is has been recognized by many top players for its difficulty. However, its survival in the voting depends on you. Check out the map and come back to decide its fate in Project Loved!
```

Comment:
Some people unfavorited, so approx to 100.
```suggestion
Known for her high difficulty and jump-based maps, Yusomi returns again with her take on Aimiya Zero's "-ERROR". The song tells a story of a character drowning from their emotions after a heartbreaking separation. Although trivial, the mapset does a clean job of simulating a similar experience. Instead of emotions, any player who attempts this map will find themselves drowning in a barrage of increasingly difficult 200 BPM jumps and bursts. I think it's a fitting theme for the song. The map itself has amassed over 100 favorites since its submission in March of 2016, and is has been recognized by many top players for its difficulty. However, its survival in the voting depends on you. Check out the map and come back to decide its fate in Project Loved!
```